operations/network/customer_accessible_networks/can_to_chn/network/chn_enable.md

```diff
@@ -154,9 +154,8 @@ Copying and storing all data in `${BACKUPDIR}` off-system in a version control r
 1. (`ncn-m001#`) Disable CFS changes on UAN to prevent migration to the CHN.
 
    ```bash
-   for xname in $(cray hsm state components list --role Application --subrole UAN --type Node --format json | jq -r .Components[].ID) ; do
-       cray cfs v3 components update --enabled false --format json "${xname}"
-   done
+   xname=`cray hsm state components list --role Application --subrole UAN --type Node --format json | jq -r '[.Components[].ID]|join(",")'`
+   cray cfs v3 components updatemany --filter-ids "${xname}" --enabled false --format json
```

Comment:
In the old way, they ended up implicitly seeing a list of all of the xnames, in the output of the CLI command. The new method will only show them the list that succeeded, so I think we should add a line to also print the entire list up front.
```suggestion
   echo "Disabling CFS on: ${xname}"
   cray cfs v3 components updatemany --filter-ids "${xname}" --enabled false --format json
```